code/framework/extension/src/main/java/io/cattle/platform/extension/impl/ExtensionManagerImpl.java

```diff
@@ -134,12 +139,24 @@ public synchronized void start() {
                 ExtensionList<?> extensionList = getExtensionListInternal(key);
                 extensionList.inner.clear();
                 extensionList.inner.addAll(getList(key));
+                ExtensionMap<String, Object> extensionMap = getExtensionMapInternal(key);
+                extensionMap.inner.clear();
+                extensionMap.inner.putAll(getMap(key));
             }
 
             started = true;
         }
     }
 
+    protected synchronized Map<String, Object> getMap(String key) {
```

Comment:
I made it synchronized as i was following the same implemntation @ibuildthecloud used for io.cattle.platform.extension.impl.ExtensionManagerImpl#getList